asciidoc/courses/genai-fundamentals/modules/1-generative-ai/lessons/1-what-is-genai/lesson.adoc

```diff
@@ -42,16 +42,16 @@ These models are trained on vast amounts of text data and can perform various ta
 [.slide.discrete]
 ## LLM Responses
 
-The response generated by an LLM is a probabilistic continuation of the instructions it receives. 
+The response generated by an LLM is a probabilistic continuation of the instructions it receives.
 
 The LLM provides the most likely response based on the patterns it has learned from its training data.
 
-If presented with the instruction: 
-    
+If presented with the instruction (or *prompt*):
```

Comment:
I avoid the use of the word prompt here because "prompt" has a specific meaning.

This section introduces the concept in the abstract, the next section takes this further to describe prompts.